test/components/DayPicker_spec.jsx

```diff
@@ -260,6 +261,20 @@ describe('DayPicker', () => {
       });
     });
 
+    describe('#componentWillReceiveProps', () => {
```

Comment:
In order to do that, we'd need to run the test suite twice - once with jsdom, and once without it - and be able to mark tests that require the DOM such that they'd be skipped when running without jsdom.

I don't yet have a clean solution for this; i think it's OK to skip those tests for now.